Added 2 iOS courses by Angela Yu and Stanford CS193p

2apreety18 opened the pull request.

ishandeveloper commented:
Hey @2apreety18 !
Thanks for creating a Pull Request.

However, I'm sorry I can't merge this PR, because as per the contributing guidelines, only PR submissions which include free resources are valid. Also, please refrain from adding multiple resources in a single pull request, so that it's easier to maintain this repo.

Please either commit the changes in this PR or close this one and create a new PR, with a free course.

Hit me up, once done and I'll merge this PR. Goodluck ! ✨

2apreety18 commented:
Hello @ishandeveloper !
I've changed everything as per your instructions : Added only one free course link "CS193p - Developing Apps for iOS " from the Stanford university. Hope it's all right now.
